### PR TITLE
Update statuses.py

### PR DESCRIPTION
**Pull request by greenbrown**
adding two more strings :)

### CODE REVIEW

File: statuses.py [hunk 1]
```diff
@@ -39,5 +39,7 @@
    "They're a 10, but they get cranky if they don't get their daily hug.",
    "If I text you \"🤗\" it means I'm sending you a great humongous hug.",
    "hey (with the intention of giving you a long tight hug because you deserve it)",
-   "hugs", 
+   "hugs",
+   "Less worry, more hugs",
+   "Retweet me for hugging me back ",
```
[the remark this answers is not in the document]
Comment (greenbrown) [reply statuses.py:44]: I intended to ask for a retweet the bot to hug the bot back (after giving lots of hugs quotes lol) - what do you think if I change into:

"Retweet this tweet for hugging me back"

Let me know if you still prefer to remove the second quote, and I'll remove it 😸

[the remark this answers is not in the document]
Comment (achasveachas) [reply statuses.py:44]: Ah, my bad, I think I misunderstood the grammar :)

How about "Retweet this tweet to hug me back."

[the remark this answers is not in the document]
Comment (greenbrown) [reply statuses.py:44]: Yay, sure I'll update it - thank you for the correction :))

**Comment by achasveachas** [comment]
Thank you very much @greenbrown!

If you have a Twitter account let me know so I can give you a shout out! 🤗

**Comment by greenbrown** [comment]
can you also add the label hacktoberfest on this PR, I am just happy to see the accepted label on hacktoberfest site lol 😹 

Yes, my handle is @21lidyaa, thank you @achasveachas 🧸

**Comment by achasveachas** [comment]
The hacktoberfest topic is added to the repo, so as long as the PR is accepted you should see it on your profile. Let me know if you don't see it there and I can add the label to the PR just to make sure.

**Pull request merged**